seed/models/building_file.py

```diff
@@ -183,47 +183,8 @@ def process(self, organization_id, cycle, property_view=None):
         else:
             property_state = self.property_state
 
-        # merge or create the property state's view
-        if property_view:
-            # create a new blank state to merge the two together
-            merged_state = PropertyState.objects.create(organization_id=organization_id)
-
-            # assume the same cycle id as the former state.
-            # should merge_state also copy/move over the relationships?
-            priorities = Column.retrieve_priorities(organization_id)
-            merged_state = merge_state(
-                merged_state, property_view.state, property_state, priorities['PropertyState']
-            )
-
-            # log the merge
-            # Not a fan of the parent1/parent2 logic here, seems error prone, what this
-            # is also in here: https://github.com/SEED-platform/seed/blob/63536e99cf5be3a9a86391c5cead6dd4ff74462b/seed/data_importer/tasks.py#L1549
-            PropertyAuditLog.objects.create(
-                organization_id=organization_id,
-                parent1=PropertyAuditLog.objects.filter(state=property_view.state).first(),
-                parent2=PropertyAuditLog.objects.filter(state=property_state).first(),
-                parent_state1=property_view.state,
-                parent_state2=property_state,
-                state=merged_state,
-                name='System Match',
-                description='Automatic Merge',
-                import_filename=None,
-                record_type=AUDIT_IMPORT
-            )
-
-            property_view.state = merged_state
-            property_view.save()
-
-            merged_state.merge_state = MERGE_STATE_MERGED
-            merged_state.save()
-
-            # set the property_state to the new one
-            property_state = merged_state
-        elif not property_view:
-            property_view = property_state.promote(cycle)
-        else:
-            # invalid arguments, must pass both or neither
-            return False, None, None, "Invalid arguments passed to BuildingFile.process()"
```

Comment:
Moving this block to _after_ we create the scenarios/measures/meters. This is essentially a reversion of building_file changes in #2117